data/core/units/elves/Enchantress.cfg

```diff
@@ -33,14 +33,15 @@
     [/attack]
     [attack]
         name=entangle
-        description=_"entangle"
+        description=_"ethereal web"
```

Comment:
Suggestion: `# po: in game mechanics an upgrade of the Shaman’s slowing “entangle”, but in lore has the different magic style of the Enchantress’s unit description`

Only needs to go into one of the units' descriptions, otherwise the .pot file will put have duplicate comments on the same entry.

Reply:
Added. Just copied your suggest hint over.